AGPs/AGP-LD1.md

```diff
@@ -0,0 +1,63 @@
+
+Quarterly transfers of 100K ANT with no expiration date and beginning after ANV 5 and due the week after each ANV.
+One time transfer of 50K DAI for initial execution and launch of the App Mining program.
+
+## Purpose of the transfer
+
+To fund an "Aragon App Mining" program which rewards developers for creating and maintaining Aragon applications. The policy is inspired by [Blockstack's App Mining program](https://app.co/mining).
+
+A core value proposition of Aragon is that it is a platform which allows users to pull from a library of modular and composable applications to create and operate an organization on-chain. In order to attract application developers to the platform we need to provide a stable opportunity that encourages entrepreneurial developers to understand user needs, develop applications, ship them to mainnet, and to continue to maintain and improve them over time.
+
+At the moment there is no incentive to do this outside of one-off grants. There has been discussion to create monetization models for the eventual release of the App Center, but currently such a system doesn't yet exist, and even if it did the number of users of Aragon is small enough that such a market may not be attractive enough to bring top talent into the space. Users will come if there are high quality apps, but if there are no users, no high quality apps will be built and maintained.
+
+To remedy this I propose we bootstrap the supply side by allocating funds and distributing them to Aragon Application developers based on agreed upon KPIs. We can combine multiple KPIs into an **Aragon App Score**, which we can use to rank applications and then use the ranking to distribute funding to maintainers on a periodic basic.
+
+**The following weighted KPIs would be used to create the Aragon App Score:**
+
+| KPI | Formula | Weight |
+| -------- | -------- | -------- |
+| Utility    | Sum of (# of transactions involving an application ) | 30%  |
+| Value weighted utility    | Sum of (# of transactions involving an application * DAI value of organization's treasury) | 30%    |
+| Popularity    | Sum of (# of installations) | 10%  |
+| Value weighted Popularity    | Sum of (# of installations *  DAI value of organization's treasury) | 30% |
+
+To compute the DAI value of an organizations treasury we would use the *default vault* and compute the DAI value of ETH, ANT, and DAI held by the organization.
+
+To compute the # of transactions involving an application we would simply examine the blockchain.   
+
+For each payout cycle, there is a determined “pot” of total earnings that will get paid to apps. The top app gets paid 20% of the total pot. So, for a pot of 100K ANT, the top app receives 20K ANT. The next app gets paid 20% of the remaining pot. The remaining pot is 80K, and 20% of that is 16K ANT. This process continues until every app has been paid or the payout amount is bellow 200 ANT whichever comes first.
```

Comment:
Let's imagine there are five apps, pot is 100k ANT as mentioned.

#1. 20k ANT
#2. 16k ANT
#3 12800 ANT
#4 10240 ANT
#5 8192 ANT

Would the balance (32768 ANT) get sent back to the AA multisig? In any case, you should mention what happens to the leftover balance of the pot.

Reply:
bump